### PR TITLE
[0.65] Drop ChakraCore dependency

### PR DESCRIPTION
**Pull request by JunielKatarn**
Backport of #9056

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/9092)

### CODE REVIEW

File: vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj [hunk 1]
```diff
@@ -159,32 +157,27 @@
...
     <Import Project="$(SolutionDir)\packages\Microsoft.SourceLink.Common.1.0.0\build\Microsoft.SourceLink.Common.targets" Condition="Exists('$(SolutionDir)\packages\Microsoft.SourceLink.Common.1.0.0\build\Microsoft.SourceLink.Common.targets')" />
     <Import Project="$(SolutionDir)\packages\Microsoft.SourceLink.GitHub.1.0.0\build\Microsoft.SourceLink.GitHub.targets" Condition="Exists('$(SolutionDir)\packages\Microsoft.SourceLink.GitHub.1.0.0\build\Microsoft.SourceLink.GitHub.targets')" />
-    <Import Project="$(SolutionDir)\packages\ReactWindows.ChakraCore.ARM64.1.11.20\build\native\ReactWindows.ChakraCore.ARM64.targets" Condition="'$(Platform)' == 'ARM64' AND Exists('$(SolutionDir)\packages\ReactWindows.ChakraCore.ARM64.1.11.20\build\native\ReactWindows.ChakraCore.ARM64.targets')" />
+    <Import Project="$(HermesPackage)\build\native\ReactNative.Hermes.Windows.targets" Condition="Exists('$(HermesPackage)\build\native\ReactNative.Hermes.Windows.targets')" />
```
Comment (NickGerleman) [inline vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj:165]: Hermes added?

File: vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj [hunk 2]
```diff
@@ -159,32 +157,27 @@
...
   <ImportGroup Label="ExtensionTargets">
     <Import Project="$(SolutionDir)packages\ReactWindows.OpenSSL.StdCall.Static.1.0.2-p.5\build\native\ReactWindows.OpenSSL.StdCall.Static.targets" Condition="Exists('$(SolutionDir)packages\ReactWindows.OpenSSL.StdCall.Static.1.0.2-p.5\build\native\ReactWindows.OpenSSL.StdCall.Static.targets')" />
-    <Import Project="$(SolutionDir)packages\boost.1.72.0.0\build\boost.targets" Condition="Exists('$(SolutionDir)packages\boost.1.72.0.0\build\boost.targets')" />
-    <Import Project="$(SolutionDir)packages\Microsoft.ChakraCore.vc140.1.11.24\build\native\Microsoft.ChakraCore.vc140.targets" Condition="Exists('$(SolutionDir)packages\Microsoft.ChakraCore.vc140.1.11.24\build\native\Microsoft.ChakraCore.vc140.targets')" />
-    <Import Project="$(SolutionDir)packages\ChakraCore.Debugger.0.0.0.44\build\native\ChakraCore.Debugger.targets" Condition="Exists('$(SolutionDir)packages\ChakraCore.Debugger.0.0.0.44\build\native\ChakraCore.Debugger.targets')" />
+    <Import Project="$(SolutionDir)packages\boost.1.76.0.0\build\boost.targets" Condition="Exists('$(SolutionDir)packages\boost.1.76.0.0\build\boost.targets')" />
```
Comment (NickGerleman) [inline vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj:160]: Can we avoid updating Boost with this change?

File: vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj (hunk 1 above)
Comment (JunielKatarn) [reply vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj:165]: Good catch. That one slipped with the merge. Reverting...

**Review by NickGerleman — APPROVED** [review]

File: vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj (hunk 2 above)
Comment (JunielKatarn) [reply vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj:160]: We can.

**Pull request merged**